File: RenataApp.API/Models/Inventory.cs
using System;

namespace RenataApp.API.Models
{
    public class Inventory
    {
        public int Id { get; set; }
        public DateTime  Date { get; set; }
        public string StoreName { get; set; }
        public string TypeName { get; set; }
        public string ModelName { get; set; }
        public decimal Cost { get; set; }
        public string Imei { get; set; }
        public bool InStock { get; set; }

        public  string User { get; set; }
        public  int UserId { get; set; } 
    }
}

File: RenataApp-SPA/src/app/_models/inventory.ts
import { User } from './user';

export interface Inventory {
    id: number;
    date: Date;
    storeName: string;
    typeName: string;
    modelName: string;
    cost: any;
    imei: string;
    inStock: boolean;
    user: User;
    userId: number;
}
